typescript/vrotest/src/build.ts

```diff
@@ -312,43 +296,69 @@ export default async function (flags: BuildCommandFlags) {
 		]);
 	}
 
-	async function createNodeProjectConfig(): Promise<void> {
-		/**
-		 * Generate package.json file in the test folder in order to:
-		 * 1. Allow tests to be run outside of Build Tools for VMware Aria.
-		 * 2. Enable resolution of .nycrc file by the code coverage CLI.
-		 */
-		const projectJson = {
-			"name": "vro-test",
-			"version": "1.0.0",
-			"description": "",
-			"scripts": {
-				"test": "jasmine --config=./jasmine.json",
-				"test:coverage": "nyc npm run test && nyc report"
-			},
-			"devDependencies": {
-				"jasmine": "^4.0.2",
-				"nyc": "^15.1.0"
-			}
-		};
-		fs.writeFile(path.join(flags.output, constants.NODE_PROJECT_CONFIG_FILE), JSON.stringify(projectJson, null, 2));
-	}
+    async function configureTestFramework(flags: BuildCommandFlags) {
+        const customTestsConfigPath = path.join(flags.projectRoot, constants.TEST_CONFIG_PATH);
 
-	async function createJasmineConfig(): Promise<void> {
-		const jasmineConfig = {
-			spec_dir: "test",
-			spec_files: ["**/?(*.)+(spec|test).[jt]s?(x)"],
-			helpers: ["../helpers/*.js"],
-			failSpecWithNoExpectations: false,
-			stopSpecOnExpectationFailure: false,
-			stopOnSpecFailure: false,
-			random: false,
-		};
-		fs.writeFile(path.join(flags.output, constants.JASMINE_CONFIG_FILE), JSON.stringify(jasmineConfig, null, 2));
-	}
+        if (fs.pathExistsSync(customTestsConfigPath)) {
+            // Use use-defined unit tests bootstrapping configuration ("unit-tests.config/*")
+            await Promise.all(fs.readdirSync(customTestsConfigPath).map(fileName => {
+                const sourceFilePath = path.join(customTestsConfigPath, fileName);
+                const destFilePath = path.join(flags.output, fileName);
+                return fs.copy(sourceFilePath, destFilePath, { overwrite: true, recursive: true, errorOnExist: false });
+            }));
+        } else {
+            // Build configuration based on the framework selection
+            const templatesPath = path.join(constants.TEST_CONFIG_TEMPLATES_PATH, flags.testFrameworkPackage || "jasmine");
+            console.log("Copying templates...");
+            console.log(templatesPath);
```

Comment:
Maybe debug logs left over? But if not we can probably touch them up a little bit

Reply:
Good catch, thanks.